integration/library_test.go

```diff
@@ -0,0 +1,216 @@
rcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "libcap", Version: "2.22", Release: "9.amzn2.0.2", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "libassuan", Version: "2.1.0", Release: "3.amzn2.0.2", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "nss-softokn", Version: "3.36.0", Release: "5.amzn2", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "libuuid", Version: "2.30.2", Release: "2.amzn2.0.4", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "libmetalink", Version: "0.1.2", Release: "7.amzn2.0.2", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "libdb-utils", Version: "5.3.21", Release: "24.amzn2.0.3", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "libunistring", Version: "0.9.3", Release: "9.amzn2.0.2", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "ncurses", Version: "6.0", Release: "8.20170212.amzn2.1.2", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "pth", Version: "2.0.7", Release: "23.amzn2.0.2", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "openssl-libs", Version: "1.0.2k", Release: "16.amzn2.1.1", Epoch: 1, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "pyliblzma", Version: "0.5.3", Release: "11.amzn2.0.2", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "libmount", Version: "2.30.2", Release: "2.amzn2.0.4", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "nss-pem", Version: "1.0.3", Release: "5.amzn2", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "libssh2", Version: "1.4.3", Release: "12.amzn2.2", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "rpm-libs", Version: "4.11.3", Release: "25.amzn2.0.3", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "rpm-build-libs", Version: "4.11.3", Release: "25.amzn2.0.3", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "python-pycurl", Version: "7.19.0", Release: "19.amzn2.0.2", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "yum-plugin-ovl", Version: "1.1.31", Release: "46.amzn2.0.1", Epoch: 0, Arch: "noarch", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}},
```

Comment:
I feel it is difficult to maintain it. How about saving these results as golden files?

Reply:
I'm OK with either approach. I personally think keeping them together in code is nice because they're still version controlled anyway so we'll see a change, although it's very unlikely that a change will happen. But I can turn them into golden files if you prefer.